core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectLifecycleSpec.scala

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.dsde.rawls.billing
+
+import org.broadinstitute.dsde.rawls.TestExecutionContext
+import org.broadinstitute.dsde.rawls.config.MultiCloudWorkspaceConfig
+import org.broadinstitute.dsde.rawls.dataaccess.SamDAO
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord.JobType.JobType
+import org.broadinstitute.dsde.rawls.model.{
+  CreateRawlsV2BillingProjectFullRequest,
+  CreationStatuses,
+  RawlsBillingProjectName,
+  RawlsRequestContext,
+  SamBillingProjectActions,
+  SamResourceTypeNames
+}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+import org.mockito.Mockito.{verify, when, RETURNS_SMART_NULLS}
+
+import java.util.UUID
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+// so we don't have to define a new type in order to call test methods in our instantiations of the interface
```

Comment:
I'm not really grokking this comment -- is it required for `exposeUnregisterBillingProject`?